net/l3roamd/Makefile

```diff
@@ -0,0 +1,28 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=l3roamd
+PKG_SOURCE_DATE:=2016-05-06
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/freifunk-gluon/l3roamd.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=9d474a9329e307a5e71fd55d53491d93d34dda13
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
+
+CMAKE_OPTIONS += -DCMAKE_BUILD_TYPE:STRING=MINSIZEREL
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/l3roamd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=The layer 3 roaming daemon
+  DEPENDS:=+libnl-tiny +kmod-tun +librt +kmod-cfg80211
```

Comment:
Then what does mmfd do for us? And how does roaming between non-neighbour nodes work?